corehq/apps/app_manager/models.py

```diff
@@ -352,11 +352,11 @@ def get_action_properties(self, action):
         action_properties = action.properties()
         if 'name_path' in action_properties and action.name_path:
             yield 'name', action.name_path
-        if action_properties.get('name_update') and action.name_update.question_path:
+        if action_properties.get('name_update') and action.name_update and action.name_update.question_path:
```

Comment:
curious why we do a `action_properties.get('name_update')` here but the other conditionals check `<property_name>` in `action_properties`. Doesn't `action_properties.get(<property_name>)` yield the same result as `action.<property_name>`, but the former is the safer route if it's missing? I guess what I'm trying to also figure out is why add the `action.name_update` here? It feels a little brittle to have very similar checks next to each other, but no comment for clarification?